src/api/getMovie.js

```diff
@@ -0,0 +1,12 @@
+import apiKey from "./apiKey";
+
+const getMovie = async (id, signal, plot = "short") => {
+  const response = await fetch(
+    `https://omdbapi.com/?apikey=${apiKey}&i=${id}&plot=${plot}`,
+    { signal }
+  );
+  const movie = await response.json();
+  return movie;
```

Comment:
I think it would make more sense to just use fetch here and leave async/await for usage of this function
Simple one liner.
```
getMovies() {
  return fetch('...').then(response => response.json());
}